spec/labor/markdown_fixer_spec.rb

```diff
@@ -7,6 +7,7 @@ def front_matter(title: "", description: "")
     <<~HEREDOC
       ---
       title: #{title}
+      published: false
```

Comment:
Suggestion: Is there a end-to-end test (or test for `fix-all`) this could be set to `Published: false` in?